test/normalization/classes/invalid.js

```diff
@@ -0,0 +1,15 @@
+/** @jsx h */
+
+export const checkSelection = false
+
+export const input = <editor>
+    <note classes={['class with spaces']}>
+        <p>Classes can&apos;t have spaces</p>
+    </note>
+</editor>
+
+export const output = <editor>
+    <note classes={['class', 'spaces', 'with']}>
```

Comment:
We can't keep them in a set as Slate's model must be JSON-compatible (or in other words: composed only of simple objects and primitive values), hence why I choose to keep classes as a sorted list.

Reply:
I was thinking about: `Array.from(new Set(classes))`

Reply:
This would be simpler, but JavaScript sets retain the order of insertions, so doing it this way would mean that the result of adding multiple classes would depend on their ordering. While this has no impact on us now, it could possibly complicate unifying state for cooperative editing. While this code is slightly more complex, I like it more because it makes data more regular.